extensions/sql-database-projects/src/sqldbproj.d.ts

```diff
@@ -37,21 +37,29 @@ declare module 'sqldbproj' {
 		/**
 		 * Adds the list of sql files and directories to the project, and saves the project file
 		 * @param list list of files and folder Uris. Files and folders must already exist. No files or folders will be added if any do not exist.
+		 * @param ignoreDuplicates Flag that indicates whether duplicate entries should be ignored.
```

Comment:
whether duplicate entries should be ignored or throw an error.

(and elsewhere)